test/e2e/utils/common.js

```diff
@@ -170,6 +170,12 @@ const testValues = {
   campLejeune: false
 };
 
+function selectDropdown(client, field, value) {
+  client.click(`select[name='${field}']`)
+    .click(`select option[value='${value}']`)
+    .keys(['\uE006']);
```

Comment:
Yes. Firefox has an issue with dropdown selections triggering events when being driven by selenium, and this is a bit of a workaround I found.